### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#136] [Ideas]
Hi, this project looks great!
Just to know, did you think about reproducing historical sets of unit icons?
I mean, as in Project44.ca, you could let the users choose if they prefer seeing STANAG or other Unit icons.
For example old unit map icons like German Wehrmacht, British Army, Russian Army, Italian Army and so on. 
Like these German Unit Icons in my old and incomplete website here: http://www.okh.it/places/TabIco.htm
Thank you for your work, it looks very promising!
Federico

**Comment by kjellmf** [comment]
Thank you for the suggestion and the link to your website! I have not thought about historical unit icons before, but allowing other icons than MILSTD2525/APP-6 is a great suggestion. [map.army](https://map.army) has emoji symbols, so adding something similar to ORBAT mapper should be doable. Do you know if some of the icons on your site are available as vector graphics (SVG)?

**Reply by federicopeyrani1969** [reply]
Hi Kjellmf,
The German Map Icons are available as a charcter set.
Here you fine done: http://www.okh.it/places/download/wwiigts_.ttf
And here another: http://www.okh.it/places/download/GermanMapSymbols.zip
Hope this is good for you!

Best regards and keep on your wonderful work!
Federico

**Reply by kjellmf** [reply]
Thank you for sharing! I've added an issue to the issue tracker (#137) so that you can keep track of the progress. I'm currently working on a new map user interface, so I don't know when I can start working on this.

**Reply by federicopeyrani1969** [reply]
You're Welcome!
I really like your work. 
When it will be ready, please let me know.

I have another question:
Will you consider importing kmz files?
I have plotted down many battles in the Med theatre, and some history units in Normandy (Tiger Battalions).
I would like to see how will they look into orbat-mapper.
If you like, I can send you some to show how they look as kmz.
Regards
Federico

**Reply by kjellmf** [reply]
Thanks! KML/KMZ import is on my TODO list (#138). I would love to see some sample KMZ files. Always nice to have something to test with. If you don't want to share the files publicly you can contact me on orbat-mapper@proton.me

**Reply by federicopeyrani1969** [reply]
Just sent you an email!
Bye 
Federico

**Reply by federicopeyrani1969** [reply]
In the next days I'll try to work on a project to give you feedback. 
It will take some time as I have not much spare time myself, but this will help me to assist you further.
Federico

**Comment by kjellmf** [comment]
Some fonts with alternative icons. Adding the link here for reference http://www.mapsymbs.com/fontstuf.html

**Comment by kjellmf** [comment]
Custom icons can now be added to a scenario (https://github.com/orbat-mapper/orbat-mapper/pull/293)
<img width="872" height="483" alt="image" src="https://github.com/user-attachments/assets/ceacd9e4-b204-4626-b48f-f6cdda24103c" />
